stellargraph/core/utils.py

```diff
@@ -123,19 +123,24 @@ def chebyshev_recurrence(T_k_minus_one, T_k_minus_two, X):
     return T_k
 
 
-def GCN_Aadj_feats_op(features, A, **kwargs):
+def GCN_Aadj_feats_op(features, A, k=1, **kwargs):
```

Comment:
Can't we just have `filter='localpool'` in the argument definition instead of `filter = kwargs.get("filter", "localpool")` later in the code?